tests/unit/layer/volumetric/cloudvol/test_backend.py

```diff
@@ -141,7 +141,7 @@ def test_cv_backend_write_scalar(clear_caches, mocker):
 
     index = VolumetricIndex(
         bcube=BoundingCube.from_slices((slice(0, 1), slice(1, 2), slice(2, 3))),
-        resolution=(1, 1, 1),
+        resolution=Vec3D((1, 1, 1)),
```

Comment:
This is pretty strange - I think I had the wrong version of the file checked out on my local branch and it never showed up on git status for some reason.

Reply:
I had the same. Then I noticed that the line numbers differ and re-cloned, and then it worked